source/fundamentals/databases-collections.txt

```diff
@@ -13,6 +13,9 @@ Databases and Collections
 Overview
 --------
 
+In this guide, you can learn how to use databases and collections in the
```

Comment:
```suggestion
In this guide, you can learn how to use MongoDB databases and collections with the
```

Suggestion: I think this could clarify the relationship between the driver and server. I think "in" should be replaced with "with" as I do not believe the driver has databases or collections.